src/main/java/io/ebeaninternal/server/deploy/BeanProperty.java

```diff
@@ -303,7 +306,8 @@ public BeanProperty(BeanDescriptor<?> descriptor, DeployBeanProperty deploy) {
     this.dbLength = deploy.getDbLength();
     this.dbScale = deploy.getDbScale();
     this.dbColumnDefn = InternString.intern(deploy.getDbColumnDefn());
-    this.dbColumnDefault = deploy.getDbColumnDefault();
+    this.dbColumnDefault = DbDefaultValue.toSqlLiteral(deploy.getDbColumnDefault(), deploy.getPropertyType(), deploy.getDbType());
```

Comment:
I think it is the best, if the value from @DbDefault is converted to a valid SQL-literal here.